client/src/modules/account_reference_type/modals/account_reference_type.modal.html

```diff
@@ -0,0 +1,47 @@
+<form name="AccountReferenceTypeForm" bh-submit="AccountReferenceTypeModalCtrl.submit(AccountReferenceTypeForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>TREE.ADMIN</li>
+
+      <li ng-if="AccountReferenceTypeModalCtrl.isCreating" class="title">
```

Comment:
It looks like the only thing that changes here is the `badge`.  Could you move the `ngIf` to put it on the badge?

```html
<label ng-if="AccountReferenceTypeModalCtrl.isCreating" class="badge badge-warning" translate>FORM.LABELS.CREATE</label>
```